database/migrations/2016_04_06_015507_create_member_custom_field_data_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateMemberCustomFieldDataTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using short method names like CreateMemberCustomFieldDataTable::up(). The configured minimum method name length is 3.](https://www.codacy.com/app/ym-huang0808/VMS/file/2452055381/issues/source?bid=2972826&fileBranchId=3248926#l13)